userspace/engine/rule_loader.cpp

```diff
@@ -347,8 +364,9 @@ static shared_ptr<ast::expr> parse_condition(
 	}
 	catch (const sinsp_exception& e)
 	{
-		throw falco_exception("Compilation error when compiling \""
-			+ condition + "\": " + to_string(p.get_pos().col) + ": " + e.what());
+		throw falco_exception("Compilation error when compiling \n"
```

Comment:
Watch out for conflicts with https://github.com/falcosecurity/falco/pull/2098.

Reply:
resolved

Reply:
This is not solved, the changes of #2098 introduced a new form of error reposting. We should throw a `rule_loader::rule_load_exception` with proper err code here.